tests/data/test_data.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+from data import HeroTranslator, MapTranslator
+
+
+def test_HeroTranslator_with_english_characters():
+    assert HeroTranslator.get_base_hero_name('Falstad') == 'Falstad'
+    assert HeroTranslator.get_base_hero_name('Zuljin') == "Zul'jin"
+    assert HeroTranslator.get_base_hero_name('Tinker') == "Gazlowe"
+    assert HeroTranslator.get_base_hero_name(u"Gazleu /// Gazlowe") == "Gazlowe"
+    assert HeroTranslator.get_base_hero_name("Gazleu /// Gazlowe") == "Gazlowe"
+    assert HeroTranslator.get_base_hero_name(u"Thrall /// Thrall") == "Thrall"
+    assert HeroTranslator.get_base_hero_name("Thrall /// Thrall") == "Thrall"
+
```

Comment:
Can you please add tests for Gul'dan, Brightwing and Lunara? The reason is the internal representation for these heroes are Guldan, FaerieDragon and Dryad respectively and I would like to test those cases.

Reply:
Done - tests pass locally